test/integration/camera_sensor_background_from_scene.cc

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
```

Comment:
2022

Reply:
Done in https://github.com/gazebosim/gz-sim/pull/1515/commits/92bab296b7fd4c2e39732e6414d45bea2fa9d2ab